lib/config-generator.js

```diff
@@ -320,20 +320,26 @@ class ConfigGenerator {
          *  * https://github.com/webpack/webpack.js.org/issues/652#issuecomment-273324529
          *  * https://webpack.js.org/guides/caching/#deterministic-hashes
          */
-        // when not using versioning, none of this is important
-        if (this.webpackConfig.useVersioning) {
-            let moduleNamePlugin;
-            if (this.webpackConfig.isProduction()) {
+        let moduleNamePlugin;
+        if (this.webpackConfig.isProduction()) {
+            // only enable if versioning is on, as it makes the assets *slightly* larger
```

Comment:
Can we have a way to turn it ON otherwise too ? If you use query-string versioning with https://packagist.org/packages/incenteev/hashed-asset-bundle, having a stable file hash is important too.